index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Crete Round" />
+        
+    </head>
+    <body>
+        <section class="section-1">
+            <nav class="nav-bar">
+                    <div class="logo-name">
+                            <p>DANIEL OCHUBA</p>
+                    </div>
+                    <div class="menu-button">
+                        <div class="toggle"></div>
+                        <div class="toggle"></div>
+                        <div class="toggle"></div>
+                    </div>
+                    
+                </nav>
+        </section>
+
+        <section>
+            <div class="section-2">
+                <h1 class="primary-text">Hey There. I’m Daniel</h1>
+                <h5 class="supporting-text">I’am a Software Developer</h5>
+                <p class="description">I can help you build a product ,
+                    feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <div class="images">
+                    <div class="icons">  
+                        <span class="icon"><img src="./Disabled(2).png" alt=""></span>
+                        <span class="icon"><img src="./Disabled(1).png" alt=""></span>
+                        <span class="icon"><img src="./Enabled.png" alt=""></span>
+                        <span class="icon"><img src="./Disabled(3).png" alt=""></span>
+                        <span class="icon"><img src="./Disabled.png" alt=""></span>
+                        
+                    </div>
```

Comment:
- [ ] KIndly make sure all social media links are wrapped into the `ul` tag and make sure every link is rendered using the Anchor tag and wrapped into the `li` tag. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using the `href` attribute.